artemis-commons/src/main/java/org/apache/activemq/artemis/utils/uri/BeanSupport.java

```diff
@@ -35,18 +35,27 @@
 public class BeanSupport {
 
    private static final BeanUtilsBean beanUtils = new BeanUtilsBean();
+   private static final Map<Converter, Class> customConverters = new HashMap<>();
 
    static {
-      // This is to customize the BeanUtils to use Fluent Proeprties as well
+      // This is to customize the BeanUtils to use Fluent Properties as well
       beanUtils.getPropertyUtils().addBeanIntrospector(new FluentPropertyBeanIntrospectorWithIgnores());
    }
 
    public static void registerConverter(Converter converter, Class type) {
       synchronized (beanUtils) {
          beanUtils.getConvertUtils().register(converter, type);
+         customConverters.put(converter, type);
       }
    }
 
+   public static void customise(BeanUtilsBean bean) {
+      for (Map.Entry<Converter, Class> existing : customConverters.entrySet()) {
+         bean.getConvertUtils().register(existing.getKey(), existing.getValue());
+      }
```

Comment:
The other customConverters usage is synchronized, so this probably should be also.

Could be nicer with:
```
      customConverters.forEach((key, value) -> {
         bean.getConvertUtils().register(key, value);
      });
```